### PR TITLE
Added Operator and CR DR Tests

### PR DESCRIPTION
**Pull request by gejain**
**What type of PR is this?**
>integration-test

**What this PR does / why we need it**:
Added Operator and CR DR Tests'

Test Job Passed Result -
https://jenkins.pwx.dev.purestorage.com/job/Users/job/Geet/job/CRTestStork/6/

### CODE REVIEW

File: test/integration_test/operator_test.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	validateAndDestroyCrMigration(t, appNameKafka, appPathKafka)
+}
+
+func validateAndDestroyCrMigration(t *testing.T, appName string, appPath string) {
+	err := setSourceKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+	appData := asyncdr.GetAppData(appName)
+	pods_created, err := asyncdr.PrepareApp(appName, appPath)
+	require.NoError(t, err, "Error creating pods")
+	pods_created_len := len(pods_created.Items)
```
Comment (Rohit-PX) [inline test/integration_test/operator_test.go:64]: camelCase

File: test/integration_test/operator_test.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+func testKafkaMig(t *testing.T) {
+	// reset mock time before running any tests
+	err := setMockTime(nil)
+	require.NoError(t, err, "Error resetting mock time")
+	validateAndDestroyCrMigration(t, appNameKafka, appPathKafka)
+}
+
+func validateAndDestroyCrMigration(t *testing.T, appName string, appPath string) {
+	err := setSourceKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
```
Comment (Rohit-PX) [inline test/integration_test/operator_test.go:60]: Add new line after every require that way the code is more readable IMO.

File: test/integration_test/operator_test.go [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+	setSourceKubeConfig()
+	mig, err := asyncdr.CreateMigration(migNamePref+appName, appData.Ns, clusterPairName, appData.Ns, &includeVolumesFlag, &includeResourcesFlag, &startApplicationsFlag)
+	err = asyncdr.WaitForMigration([]*storkv1.Migration{mig})
+	require.NoError(t, err, "Error waiting for migration")
+	// As apps take time to come up on destination, putting some wait time here
+	time.Sleep(5 * time.Minute)
+	setDestinationKubeConfig()
+	pods_migrated, err := core.Instance().GetPods(appData.Ns, nil)
+	require.NoError(t, err, "Error getting migrated pods")
+	pods_migrated_len := len(pods_migrated.Items)
```
Comment (Rohit-PX) [inline test/integration_test/operator_test.go:85]: camel case

File: test/integration_test/operator_test.go (hunk 2 above)
Comment (gejain) [reply test/integration_test/operator_test.go:63]: Done.

File: test/integration_test/operator_test.go (hunk 3 above)
Comment (gejain) [reply test/integration_test/operator_test.go]: Done

File: test/integration_test/operator_test.go (hunk 1 above)
Comment (gejain) [reply test/integration_test/operator_test.go]: Done

**Review by Rohit-PX — APPROVED** [review]
LGTM.

**Comment by gejain** [comment]
Closing as Fork is not getting merged.
Opened - https://github.com/libopenstorage/stork/pull/1497

**Pull request closed without merging**